pkg/auth/native_store.go

```diff
@@ -30,6 +30,7 @@ func GetCredentialsStore(registryHostname string) (CredentialStore, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to load config file, error: %v", err)
 	}
+	fmt.Printf("test: configFile: %+v, %s", configFile, registryHostname)
```

Comment:
Looks like this is a test log? We should remove it.

Reply:
Deleted.